gprofiler/python.py

```diff
@@ -56,7 +56,7 @@ def _make_command(self, pid: int, output_path: str):
         ]
 
     def _profile_process(self, process: Process):
-        logger.info(f"Profiling process {process.pid} ({process.cmdline()})")
+        logger.info(f"Profiling process {process.pid} ({process.cmdline()})", no_server_log=True)
```

Comment:
I think that keeping the "Profiling process {pid}" is nice, so let's separate to 2 logs? 1 sent w/o commandline, 1 not sent w/ commandline.
The one sent - can include `comm` you have below.

Also - `ruby.py` has a similar snippet with `process.cmdline()`, do the same there

Reply:
Done